### PR TITLE
Feature/multi db

### PR DESCRIPTION
**Pull request by ahelal**

### CODE REVIEW

**Comment by zeelax** [comment]
+1 looks ok for me

File: defaults/main.yml [hunk 1]
```diff
@@ -8,36 +8,19 @@
...
-aws_rds_wait_timeout             : 900
-
-#aws_rds_vpc_subnet
+aws_rds_region                  : "{{ aws_region | default('eu-west-1') }}"
+aws_rds_wait                    : 900
+aws_rds_async_retries           : 100
+
+aws_rds                         :
+        instance_name           :  "{{ inventory_hostname.split('0')[0]}}"
+        db_name                 : "{{ xxx_db1"
```
Comment (asemt) [inline defaults/main.yml]: Missing `}}`?

File: tests/integeration/default/default.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+- name: Provision an ES Service domain for Ansible logging output
```
Comment (asemt) [inline tests/integeration/default/default.yml]: Copy & Paste

**Comment by asemt** [comment]
Looks good.
But please see my line comments.

**Comment by ahelal** [comment]
+1

**Pull request merged**